### ISSUE TITLE
System aggregation issue in Xamarin.iOS

### ISSUE THREAD

**Issue by rpundlik** [xamarin/xamarin-macios#3838]
### Steps to Reproduce
Its an internal application. The error occurs on on some data entry page, listing pages.

### Expected Behavior
App should not crashed
### Actual Behavior
App is getting crashed
### Environment
iOS 11.2.6

1.
Visual Studio: 2017
Visual Studio for Mac: 2017

### Description
After deploying the app on App store we are getting the below error constantly and app is getting crashed. Below is the error.

> Terminating app due to uncaught exception 'System.AggregateException', reason: 'System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. ---> System.ObjectDisposedException: Cannot access a disposed object. Object name: 'MobileAuthenticatedStream'. at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () <0x1030e7990 + 0x00028> in :0 at Mono.Net.Security.MobileAuthenticatedStream+d__58.MoveNext () <0x10341f300 + 0x005ff> in <24ab45aca3bd43dca88a4166f90ed49f#ccefd9b888727970fe79a7fe74005423>:0 --- End of inner exception stack trace --- ---> (Inner Exception #0) System.ObjectDisposedException: Cannot access a disposed object. Object name: 'MobileAuthenticatedStream'. at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () <0x1030e7990 + 0x00028> in :0 at Mono.Net.Security.MobileAuthenticatedStream+d__58.MoveNext () <0x10341f300 + 0x005ff> in <24ab45aca3bd43dca88a4166f90ed49f#ccefd9b888727970fe79a7fe74005423>:0 <--- '

The users which have this issue they are using iPhone 8 plus with OS version 11.2.6. However we are not able to get this issue on previous versions of iPhone.

Does this error related to device OR xamarin ?

Any help on this appreciated !

**Comment by therealjohn** [comment]
@rpundlik 

This looks similar to an older issue filed on [Bugzilla](https://bugzilla.xamarin.com/show_bug.cgi?id=61002#c50). Can you share your complete version information from VS for Mac to verify you are using >= Xamarin.iOS 11.6.1.4? 

The easiest way to get exact version information:
- On Visual Studio for Mac: "Visual Studio" menu, "About Visual Studio" item, "Show Details" button.
Then copy/paste the version information (you can use the "Copy Information" button).

**Comment by rpundlik** [comment]
=== Visual Studio Community 2017 for Mac ===

Version 7.0.1 (build 24)
Installation UUID: ebd1f27a-88ad-4da4-9d77-94d01a32bb9a
Runtime:
	Mono 5.4.1.7 (2017-06/e66d9abbb27) (64-bit)
	GTK+ 2.24.23 (Raleigh theme)

	Package version: 504010007

=== NuGet ===

Version: 4.0.0.2323

=== .NET Core ===

Runtime: Not installed
SDK: Not installed
MSBuild SDKs: /Library/Frameworks/Mono.framework/Versions/5.4.1/lib/mono/msbuild/15.0/bin/Sdks

=== Xamarin.Profiler ===

Version: 1.6.0
Location: /Applications/Xamarin Profiler.app/Contents/MacOS/Xamarin Profiler

=== Xamarin.Android ===

Not Installed

=== Apple Developer Tools ===

Xcode 8.3.2 (12175)
Build 8E2002

=== Xamarin.iOS ===

Version: 11.6.1.3 (Visual Studio Community)
Hash: f70a1348
Branch: xcode9.2
Build date: 2017-12-18 14:47:16-0500

=== Xamarin.Mac ===

Version: 4.0.0.215 (Visual Studio Community)

=== Xamarin Inspector ===

Version: 1.3.2
Hash: 461f09a
Branch: 1.3-release
Build date: Tue, 03 Oct 2017 18:26:57 GMT
Client compatibility: 1

=== Build Information ===

Release ID: 700010024
Git revision: 7ab1ca2ced6f584e56b7a0d4d321d00775cd95c9
Build date: 2017-05-19 05:44:51-04
Xamarin addins: 08d17158f3365beee5e60f67999e607cce4b3f93
Build lane: monodevelop-lion-d15-2

=== Operating System ===

Mac OS X 10.12.4
Darwin 16.5.0 Darwin Kernel Version 16.5.0
    Fri Mar  3 16:52:33 PST 2017
    root:xnu-3789.51.2~3/RELEASE_X86_64 x86_64

**Comment by VincentDondain** [comment]
Bonjour,

As @therealjohn said this issue is a duplicate of https://bugzilla.xamarin.com/show_bug.cgi?id=61002. Also your version information points to XI 11.6.1.3 instead of 11.6.1.4, see https://releases.xamarin.com/service-release-15-5-5.

Please update to the **latest version** of Xamarin.iOS in order to get the fix.

I'll close this bug since it's a duplicate, feel free to reopen if you're still experiencing the issue after updating.

_Note: for Android related issue please go to https://github.com/xamarin/xamarin-android/issues._
